filament/src/materials/colorGrading/colorGradingAsSubpass.mat

```diff
@@ -6,6 +6,11 @@ material {
             name : lut,
             precision: medium
         },
+        {
+            type : sampler2d,
+            name : lut1d,
+            precision: medium
+        },
```

Comment:
I would prefer that you still use a sampler3d (so we don't have to have two samplers), just create a Nx1x1 texture. Wouldn't that make things simpler?